examples/gwlb_with_vmseries/variables.tf

```diff
@@ -112,10 +112,10 @@ variable "gateway_load_balancers" {
   - `name`                - (required|string) Gateway Load Balancer name.
   - `vnet_key`            - (required|string) Key of a VNet from `var.vnets` that contains target Subnet for LB's frontned. Used to get Subnet ID in combination with `subnet_key` below.
   - `subnet_key`          - (required|string) Key of a Subnet from `var.vnets[vnet_key]`.
-  - `frontend_ip_config`  - (optional|map) Remaining Frontned IP configuration.
+  - `frontend_ip`         - (optional|map) Remaining Frontned IP configuration.
   - `resource_group_name` - (optional|string) LB's Resource Group, by default the one specified by `var.resource_group_name`.
   - `backends`            - (optional|map) LB's backend configurations.
-  - `heatlh_probe`        - (optional|map) Health probe configuration.
+  - `health_probes`       - (optional|map) Health probes configuration.
 
   Please consult [module documentation](../../modules/gwlb/README.md) for details.
   EOF
```

Comment:
missing type definition

Reply:
Example is going to be delivered in new PR.